src/EntityFrameworkCore.Projectables/Services/ProjectionExpressionResolver.cs

```diff
@@ -15,11 +15,14 @@ public LambdaExpression FindGeneratedExpression(MemberInfo projectableMemberInfo
 
             var expression = GetExpressionFromGeneratedType(projectableMemberInfo);
 
+            if (expression is null && projectableAttribute.UseMemberBody is not null && projectableAttribute.MemberBodyParameterValues is null)
+            {
+                expression = GetExpressionFromGeneratedType(projectableMemberInfo, true, projectableAttribute.UseMemberBody);
```

Comment:
The result will be overwritten by the call on line 24

Reply:
I think differently that when calling line 20 the expression will not be empty